scripts/vcf2hdf5_parallel

```diff
@@ -253,6 +253,17 @@ def Main(argv=None):
                     chrom = fields[args.regions_column].split(':')[0]
                     if chrom not in chrom_list:
                         chrom_list[chrom] = True
+
+    # check chrom_list for invalid chars '_' and ':'
+    # @TCC at least '_' could probably be allowed if vcf2npy output file nameing was changed (need a separator between chrom and position range)
+    for c in chrom_list:
+        if '_' in c:
+            logging.critical("character '_' in chrom name '{}' will probably break this script".format(c))
+            sys.exit(2)
+        if ':' in c:
+            logging.critical("character ':' in chrom name '{}' will probably break this script".format(c))
+            sys.exit(2)
+             
```

Comment:
The only situation I can imagine where the presence of a '_' character in chrom name would cause a problem is if there were two chroms named like 'foo' and 'foo_bar'. This seems unlikely? 

Maybe better issue a warning but don't sys.exit, rather than bail out on any chroms containing '_' which in general should be processed fine. E.g., in P. falciparum, chroms are named like Pf3D7_01_v3, Pf3D7_02_v3, etc., which should be fine.